services/QuillLMS/db/migrate/20220418234207_create_sales_form_submissions.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class CreateSalesFormSubmissions < ActiveRecord::Migration[5.1]
+  def change
+    create_table :sales_form_submissions do |t|
+      t.string :first_name, null: false
+      t.string :last_name, null: false
+      t.string :email, null: false
+      t.string :phone_number, null: false
+      t.string :zipcode, null: false
+      t.string :collection_type, null: false
+      t.string :school_name, null: false
+      t.string :district_name, null: false
+      t.integer :school_premium_count_estimate, null: false
```

Comment:
For the three integer fields I might opt for a stronger condition: `default: 0`.  This makes constructing valid objects easier since it's no longer necessary to specify those values, while still preserving `null: false`.